### ISSUE TITLE
Style

### ISSUE THREAD

**Issue by MIJOTHY** [rjmk/prerequisites#2]
If it's kept up-to-date we could mention [our preferred coding style](https://github.com/foundersandcoders/styleguide) so that they might start getting used to it

**Comment by ronanyeah** [comment]
+1

**Comment by ronanyeah** [comment]
Or we could just go with an existing style guide. I've been using the [AirBnb](https://github.com/airbnb/javascript) one recently to keep myself in line.

**Comment by MIJOTHY** [comment]
Yeah I like the AirBnB guide - though it'd be more pro to have our own. Still, if we never get around to making it, then I'd be down for using another one, or even forking ABnB and adapting it if needs be.
